core/src/main/java/com/graphhopper/util/details/IntDetails.java

```diff
@@ -15,27 +15,33 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package com.graphhopper.reader.dem;
+package com.graphhopper.util.details;
 
-import com.graphhopper.routing.util.DataFlagEncoder;
-import com.graphhopper.storage.GraphHopperStorage;
+import com.graphhopper.routing.profiles.IntEncodedValue;
 import com.graphhopper.util.EdgeIteratorState;
 
-/**
- * Elevation interpolator for tunnels. Estimates elevations of inner nodes of
- * the tunnel based on elevations of entry/exit nodes of the tunnel.
- *
- * @author Alexey Valikov
- */
-public class TunnelElevationInterpolator extends AbstractEdgeElevationInterpolator {
+public class IntDetails extends AbstractPathDetailsBuilder {
+
+    private final IntEncodedValue ev;
+    private int intVal = -1;
 
-    public TunnelElevationInterpolator(GraphHopperStorage storage,
-                                       DataFlagEncoder dataFlagEncoder) {
-        super(storage, dataFlagEncoder);
+    public IntDetails(String name, IntEncodedValue ev) {
```

Comment:
Why is there no return minus option here?

Reply:
Do you mean why it is not possible to store negative values?

Reply:
The DecimalDetails have the option `returnMinus`. The IntDetails don't have that option. I was wondering why.